app/src/main/res/values-fr/strings.xml

```diff
@@ -9,8 +9,8 @@
         <item name="valor">"Bravoure"</item>
         <item name="instinct">"Intuition"</item>
     </string-array>
-    <string name="place1">"Dans l\'ensemble, ton Pokémon&#8230;"</string>
-    <string name="place2">"Ses stats&#8230;"</string>
+    <string name="place1">Dans l\'ensemble, ton Pokémon&#8230;</string>
+    <string name="place2">Ses stats&#8230;</string>
```

Comment:
@DJCrapsody This page https://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling says we can choose to use the quotes or the \ but there is no need for both, so you could just remove the \ as you said.